packages/edit-site/src/components/navigation-sidebar/navigation-panel/index.js

```diff
@@ -48,7 +49,7 @@ const NavigationPanel = ( { isOpen } ) => {
 			editedPostId: getEditedPostId(),
 			editedPostType: getEditedPostType(),
 			activeMenu: getNavigationPanelActiveMenu(),
-			siteTitle: siteData.name,
+			siteTitle: siteData.name && decodeEntities( siteData.name ),
```

Comment:
Maybe we should decode the site title before rendering. I think this a pattern generally used in the codebase.

https://github.com/WordPress/gutenberg/blob/28db73d54bf30fc73886e267174aee6343a01e8c/packages/edit-site/src/components/navigation-sidebar/navigation-panel/index.js#L98

Reply:
Sure. Lets use the same pattern that is used elsewhere. I will update.